nominal/core/client.py

```diff
@@ -963,6 +965,36 @@ def get_data_review(self, rid: str) -> DataReview:
         response = self._clients.datareview.get(self._clients.auth_header, rid)
         return DataReview._from_conjure(self._clients, response)
 
+    def create_event(
+        self,
+        name: str,
+        asset_rids: list[str],
+        start: datetime | IntegralNanosecondsUTC,
+        duration: timedelta,
+        type: EventType,
+        *,
+        properties: Mapping[str, str] = {},
+        labels: list[str] = [],
+    ) -> Event:
+        response = self._clients.event.create_event(
+            self._clients.auth_header,
+            event.CreateEvent(
+                name=name,
+                asset_rids=asset_rids,
+                timestamp=_SecondsNanos.from_flexible(start).to_api(),
+                duration=_to_api_duration(duration),
+                origins=[],
+                properties=dict(properties),
+                labels=labels,
+                type=type._to_api_event_type(),
+            ),
+        )
+        return Event._from_conjure(self._clients, response)
+
+    def get_events(self, uuids: list[str]) -> list[Event]:
```

Comment:
does this need UUIDs or RIDs? Generally UUIDs shouldn't be user-facing at all, only RIDs afaik

Reply:
Also, we should make less guarantees about the return type and allow a wider arg type (i.e., we don't need a list in particular, so we shouldn't ask for one)

Reply:
Events don't appear to have RIDs: https://docs.nominal.io/api-reference/event/get-events